addons/sourcemod/scripting/SprayManager.sp

```diff
@@ -2908,7 +2905,7 @@ void AdminForceSprayNSFW(int client, int target)
 		"INSERT INTO `spraynsfwlist` (`sprayhash`, `sprayersteamid`, `setbyadmin`) VALUES ('%s', '%s', '%d') \
 		ON DUPLICATE KEY UPDATE `sprayhash` = '%s', `sprayersteamid` = '%s', `setbyadmin` = '%d';",
 		g_sSprayHash[target], sAuthID[target], 1,
-		g_sSprayHash[client], sAuthID[client], 1
```

Comment:
Why change to target here ?

Reply:
Because on duplicate key update it insert the data from the client (admin) instead of the target.

Reply:
ah ok i read too fast